components/Layout.tsx

```diff
@@ -22,6 +22,7 @@ const Layout: FunctionComponent = ({ children }) => {
 
   useEffect(() => {
     Avo.docsPageViewed({
+      userId_: 'ok',
```

Comment:
Nice! Does this identify the user? 🤔

Reply:
No it won't 🤠 I don't really understand why the codegen is including the userId in the docsPageViewed

Reply:
See my comment above